static/elements/chromedash-dialog.js

```diff
@@ -0,0 +1,159 @@
+import {LitElement, css, html} from 'lit-element';
+import '@polymer/iron-icon';
+
+// This is a simplfied version of chops-dialog:
+// https://source.chromium.org/chromium/infra/infra/+/main:appengine/monorail/static_src/elements/chops/chops-dialog/chops-dialog.js
+
+class ChromedashDialog extends LitElement {
+  static get properties() {
+    return {
+      opened: {type: Boolean, reflect: true},
+    };
+  }
+
+  static get styles() {
+    return css`
+      :host {
+        position: fixed;
+        z-index: 9999;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        overflow: auto;
+        background-color: rgba(0,0,0,0.4);
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      :host(:not([opened])), [hidden] {
```

Comment:
do we meant to `:host[hidden]`, or intentionally set a global [hidden]?

Reply:
I guess that was meant to be global because a lot of the chops components have that.  However, we don't need it.  A dialog box should just be closed, not hidden.  I removed it.